packages/openapi-schema/schema.yml

```diff
@@ -1000,6 +1000,46 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GetDocumentDetailResponse'
+  /knowledge/document/export-document:
```

Comment:
The API url is not concise, you should consider `/knowledge/document/export`. Don't forget to change the path in the `KnowledgeContrller`.